Makefile

```diff
@@ -1,38 +1,84 @@
-export KO_DATA_PATH=$(shell pwd)/cmd/manager/kodata
+MODULE   = $(shell env GO111MODULE=on $(GO) list -m)
+DATE         ?= $(shell date +%FT%T%z)
+KO_DATA_PATH  = $(shell pwd)/cmd/$(TARGET)/kodata
+TARGET        = kubernetes
 
-.PHONY: all
-all: local-dev
+GOLANGCI_VERSION  = v1.30.0
 
-.PHONY: clean
-clean:
-	-ko delete -f config/
+BIN      = $(CURDIR)/.bin
+
+GO           = go
+TIMEOUT_UNIT = 5m
+TIMEOUT_E2E  = 20m
+V = 0
+Q = $(if $(filter 1,$V),,@)
+M = $(shell printf "\033[34;1m🐱\033[0m")
+
+export GO111MODULE=on
+
+$(BIN):
+	@mkdir -p $@
+$(BIN)/%: | $(BIN) ; $(info $(M) building $(PACKAGE)…)
+	$Q tmp=$$(mktemp -d); \
+	   env GO111MODULE=off GOPATH=$$tmp GOBIN=$(BIN) $(GO) get $(PACKAGE) \
+		|| ret=$$?; \
+	   rm -rf $$tmp ; exit $$ret
+
+KO = $(or ${KO_BIN},${KO_BIN},$(BIN)/ko)
+$(BIN)/ko: PACKAGE=github.com/google/ko/cmd/ko
+
+KUSTOMIZE = $(or ${KUSTOMIZE_BIN},${KUSTOMIZE_BIN},$(BIN)/kustomize)
+$(BIN)/kustomize: | $(BIN) ; $(info $(M) getting kustomize)
+	@curl -sSfL https://raw.githubusercontent.com/kubernetes-sigs/kustomize/master/hack/install_kustomize.sh | bash
+	@mv ./kustomize $(BIN)/kustomize
```

Comment:
For what is worth, I do have the following in my `.envrc` (using [direnv](https://direnv.net/))

```
export KO_BIN=$(which ko)
export GOLANGCILINT_BIN=$(which golangci-lint)
export KUSTOMIZE_BIN=$(which kustomize)
```